chia-bls/src/gtelement.rs

```diff
@@ -19,7 +19,8 @@ use pyo3::exceptions::PyValueError;
 #[cfg(feature = "py-bindings")]
 use pyo3::{pyclass, pymethods, IntoPy, PyAny, PyObject, PyResult, Python};
 
-#[cfg_attr(feature = "py-bindings", pyclass, derive(PyStreamable, Clone))]
+#[cfg_attr(feature = "py-bindings", pyclass, derive(PyStreamable))]
+#[derive(Clone)]
```

Comment:
I'm puzzled as to why this might be necessary.

Reply:
We want to Clone the GTElement in Rust so we move it out of the py-binding exclusive derives